src/main/java/net/sacredlabyrinth/phaed/simpleclans/conversation/CreateRankNamePrompt.java

```diff
@@ -31,8 +32,14 @@
             return new MessagePromptImpl(ChatColor.RED + lang("rank.already.exists", player), this);
         }
 
-        clan.createRank(rank);
-        plugin.getStorageManager().updateClan(clan, true);
-        return new MessagePromptImpl(ChatColor.AQUA + lang("rank.created", player));
+        CreateRankEvent event = new CreateRankEvent(player, clan, clan.getRank(rank));
```

Comment:
Call the event

Reply:
Is this will call?
I think it's not because of `return`

Reply:
You can pass the rank name to the event.